VideoLocker/res/layout/view_progress_download_btn.xml

```diff
@@ -15,16 +14,9 @@
         android:layout_width="25dp"
         android:layout_height="25dp"
```

Comment:
At some point these sizes should be standardized, both here and in the course outline list items.